src/sentry/utils/snuba.py

```diff
@@ -723,7 +723,7 @@ def _apply_cache_and_build_results(
     results = []
 
     if use_cache:
-        cache_keys = [get_cache_key(query_params) for _, query_params in query_param_list]
+        cache_keys = [get_cache_key(query_params[0]) for _, query_params in query_param_list]
```

Comment:
I am not confident about this change. On the one hand, `query_params[1]` and `query_params[2]` seem to always have been encoded as `"<function>"`, so they did not add any information to the cache key. On the other hand, probably want to get different results from cache when the translators differ?

Reply:
I can't see a reason why the translators would make a difference to the cache key, since that is supposed to be based on the query itself. Leaving these off makes sense to me.

Reply:
OK cool. Once deployed, I guess there will be a spike in snuba queries because the new cache keys are not populated with values yet. I hope that's OK?

Reply:
I'll keep an eye on it.